packages/manager/core/application/src/hooks/useFeatureAvailability.ts

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from '@tanstack/react-query';
+import { apiClient } from '@ovh-ux/manager-core-api';
+
+export const featureAvailabilityBaseQueryKey = 'feature-availability-';
+
+export type UseFeatureAvailabilityParams = {
+  applicationName: string;
+  featureList?: string[];
+};
+
+const getFeaturesListToCheck = ({
+  applicationName,
+  featureList = [],
+}: UseFeatureAvailabilityParams) =>
+  [applicationName].concat(
+    featureList.map((name) => `${applicationName}:${name}`),
+  );
+
+/**
+ * @example
+ * const {data, error, loading} = useFeatureAvailability(
+ *   {applicationName: 'dedicated-nasha', featureList: ['feature1', 'feature2']},
+ *   {applicationName: 'app2'},
+ *   {applicationName: 'app3', featureList: ['feat1']}
+ * );
+ * const isAppAvailable = data.['dedicated-nasha'];
+ * const isFeature1Available = data.feature1;
+ * const isFeature2Available = data.feature2;
+ * const isApp2Available = data.app2;
+ * const isApp3Available = data.app3;
+ * const isApp3Feature1Available = data.feat1;
+ */
+export const useFeatureAvailability = (
+  ...appFeatureList: UseFeatureAvailabilityParams[]
+) => {
+  const featuresListToCheck = appFeatureList.flatMap(getFeaturesListToCheck);
+
+  const fetchFeatureAvailabilityData = async () => {
+    const result = await apiClient.aapi.get(
+      `/feature/${featuresListToCheck.join(',')}/availability`,
+    );
+    return result.data || {};
+  };
```

Comment:
Using the property `applicationName` can be misleading because we have other features like `advices` `liveChat` which are not full-fledged applications but rather a parent feature.

Reply:
renamed to `parent`